ruff_lsp/server.py

```diff
@@ -1095,15 +1139,17 @@ async def _run_check_on_document(
     # If we're trying to run a single rule, add it to the command line, and disable
     # all other rules (if the Ruff version is sufficiently recent).
     if only:
-        if _executable_version(executable) >= "0.0.198":
+        if VERSION_REQUIREMENT_ALL_SELECTOR.contains(
```

Comment:
@konstin -- I re-added this, since we now allow linter versions earlier than 0.0.198.